contracts/Core/StakeManager.sol

```diff
@@ -292,20 +292,28 @@ contract StakeManager is Initializable, ACL, StakeStorage, StateManager, Pause,
         address bountyHunter
     ) external override onlyRole(STAKE_MODIFIER_ROLE) returns (uint32) {
         uint256 _stake = stakers[stakerId].stake;
-        uint256 slashPenaltyAmount = (_stake * parameters.slashPenaltyNum()) / parameters.slashPenaltyDenom();
-        _stake = _stake - slashPenaltyAmount;
-        // prettier ignore
-        uint256 bounty = (slashPenaltyAmount * parameters.bountyNum()) / parameters.bountyDenom();
 
-        if (bounty == 0) return 0;
+        uint256 bounty;
+        uint256 amountToBeBurned;
+        uint256 amountToBeKept;
+
+        // Done for stack too deep issue
```

Comment:
Pls describe the issue a bit more

Reply:
sure : https://medium.com/1milliondevs/compilererror-stack-too-deep-try-removing-local-variables-solved-a6bcecc16231

Reply:
basically there is limit upto which stack can store variables
its I think 16 at a moment 
so by creating struct, its stored in memory 
and its ref is in stack 
so it consumes only 1 place

Reply:
I meant in the comment, for easy reference later on

Reply:
oh okay

Reply:
done https://github.com/razor-network/contracts/pull/471/commits/f7f32af3fa82f9dafca4e3750054cf79453d2d8b